api-v2/models/models.py

```diff
@@ -1,37 +1,43 @@
 import os
 import psycopg2
 import config
-from werkzeug.security import generate_password_hash, check_password_hash
+from flask import jsonify
+from flask_jwt_extended import get_jwt_identity
+from werkzeug.security import generate_password_hash
 
-class Database:
-    def __init__(self, app=None):
-        dbe= config.Config.db
-        self.conn=psycopg2.connect(dbname=dbe["DATABASE_NAME"], user=dbe["DATABASE_USER"],password=dbe["DATABASE_PASS"],host=dbe["DATABASE_HOST"])        
-        self.cur = self.conn.cursor()
-
-    def commit(self):
-        self.conn.commit()
-
-    def cursor(self):
-        return self.cur
-    
-
-database = Database()
+def dbconn():
+    dbe= config.Config.db
+    conn=psycopg2.connect(dbname=dbe["DATABASE_NAME"], user=dbe["DATABASE_USER"],password=dbe["DATABASE_PASS"],host=dbe["DATABASE_HOST"])
+    return conn    
 
 class User():
 
-    def __init__(self, fname, lname,email,password):
-        super().__init__()
-        self.fname = fname
-        self.lname = lname
-        self.email = email
-        self.password = password
+    def __init__(self, fname=None, lname=None,email=None,password=None):
+        # super().__init__()
+        if fname and lname and email and password:
```

Comment:
this is not enough validation. You understand that if any of the above e.g `fname` is empty string the check will still pass right??

Reply:
Yes. Am currently working on it.